crates/prune/prune/src/segments/account_history.rs

```diff
@@ -1,9 +1,12 @@
+use std::collections::HashMap;
+
```

Comment:
thanks boss

Reply:
if you don't leave the white space, then `std` and `core` libs are ordered alphabetically with rest of deps. it's more mature rust to put std and core deps up top, for developers who care ab std/no-std code - which includes us since we included the no-std option in several crates, like `reth-evm`

Reply:
hmm I disagree, I also prefer to have all imports in one place ordered alphabetically.

> for developers who care ab std/no-std code

`#![no_std]` is exists for this reason, and developers shouldn't judge the crate `no_std` guarantees purely by deps it uses

Reply:
we really seem to not have any consistency with it 😅
```console
➜  reth git:(alexey/history-pruning-perf) ✗ rg --multiline --type rust 'use std.*;\nuse' | wc -l
     204
➜  reth git:(alexey/history-pruning-perf) ✗ rg --multiline --type rust 'use std.*;\n\nuse' | wc -l
     132
```

Reply:
using miri as a reference https://github.com/rust-lang/miri/blob/521422adc7404ed5236cf2ad801ec8d1ea3b8818/cargo-miri/src/util.rs#L1-L13

Reply:
it doesn't look like a rule that everyone follows even in core Rust team https://github.com/rust-lang/rust/blob/59a4f02f836f74c4cf08f47d76c9f6069a2f8276/compiler/rustc_ast/src/ast.rs#L21-L42

anyway, there are no `std` import in this file now, and I believe with https://github.com/paradigmxyz/reth/pull/9141#issuecomment-2198218794 we will have this as a fmt rule